akka-aeron/src/main/resources/reference.conf

```
@@ -0,0 +1,23 @@
+akka.aeron {
+  # Connnection settings for Aeron
+  connection {
+    # Timeout for communication with the local media driver. If exceeded the connection
+    # will be permanently failed and any streams created will be failed
+    driver-timeout = 10s
+    # If set to "<embedded>" an emebedded media driver will be started
+    # during connection creation. Set to the aeron directory location if
+    # using an external media driver
+    aeron-directory = "<embedded>"
+
+  }
+
+  # Configuration for running an embedded media driver
+  media-driver {
+
+  }
+
+  # Low level settings for tuning Aeron threading model
+  advanced {
```

What's the rationale behind separating these as opposed to say this, and are these not applicable to inside the `media-driver`? Just checking.
```suggestion
  threading {
    some-logical = defaults
```